File: lab9/dcarreto9.java
//Dominic Carreto
//Java Code for Lab 9

import java.util.Scanner;
import java.util.Random;

public class dcarreto9
{
    public static void main(String[] args)
    {
        Scanner keyboard = new Scanner(System.in);
        Random rand = new Random();
        int randNumber = rand.nextInt(1, 21);
        System.out.println("Guess a number between 1 and 20:");
        int userGuess = keyboard.nextInt();
        while(userGuess < 1 || userGuess > 20)
        {
            System.out.println("You input a number that is out of range (1-20) re-enter a valid number.");
            userGuess = keyboard.nextInt();
        }
        while(userGuess != randNumber)
        {
            if(userGuess > randNumber)
            {
                System.out.println("Too high!");
            }
            else
            {
                System.out.println("Too low!");
            }
            System.out.println("Guess again:");
            userGuess = keyboard.nextInt();
        }
        for (int i = 0; i < userGuess ; i++)
        {
            System.out.println("You won and guessed the number correctly!");
        }
    }
}

File: lab9/dcarreto9.c
// Dominic Carreto
// Lab 9

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
    srand(time(NULL));
    int userGuess = 0;
    int randNumber = rand() % 20 + 1;
    //random number between 1 - 20
    printf("Guess a number between 1 and 20:\n");
    scanf("%d", &userGuess);
    while (userGuess < 1 || userGuess > 20)
    {
        printf("You input a number that is out of range (1-20) re-enter a valid number.\n");
        scanf("%d", &userGuess);
    }
    while(userGuess != randNumber)
    {
        if(randNumber > userGuess)
        {
            printf("Too Low!\n");
        }
        else
        {
            printf("Too High!\n");
        }
        printf("Guess again:\n");
        scanf("%d", &userGuess);
    }
    for (int i = 0; i < userGuess ; i++)
    {
        printf("You won and guessed the number correctly!\n");
    }
    return EXIT_SUCCESS;
}
